ml-agents/mlagents/trainers/torch/components/bc/module.py

```diff
@@ -26,6 +26,7 @@ def __init__(
             for the pretrainer.
         """
         self.policy = policy
+        self.action_spec = policy.action_spec
```

Comment:
Not sure why we need a self.action_spec if the policy already has action_specs

Reply:
+1, probably better to just do `self.policy.action_spec`